### PR TITLE
Nightmare VIA support

### PR DESCRIPTION
**Pull request by quark-works**
Included .json for VIA and keymap foldear ready to compile

### CODE REVIEW

**Comment by cfbender** [comment]
Hi @Armastardo, sorry I'm just now seeing this - I'll take a closer look tomorrow. This may be something better PR'd into the Via keyboards repo however

**Comment by quark-works** [comment]
Hi. I just did both the VIA and QMK PR. Felt like I had to upload it here first.
https://github.com/the-via/keyboards/pull/203
https://github.com/qmk/qmk_firmware/pull/9672
Thanks for this amazing keyboard!

File: nightmare/firmware/qmk_firmware/keyboards/nightmare/keymaps/via/nightmare.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nightmare,
+  "vendorId": "0xB00B",
+  "productId": "0x0001",
```
Comment (cfbender) [inline nightmare/firmware/qmk_firmware/keyboards/nightmare/keymaps/via/nightmare.json]:
```suggestion
  "vendorId": "0x434B",
  "productId": "0x4E49",
```

Comment (cfbender) [reply nightmare/firmware/qmk_firmware/keyboards/nightmare/keymaps/via/nightmare.json]: Same as other PRs!

**Review by cfbender — CHANGES_REQUESTED** [review]
Thanks again for this and for reminding me to get moving on standardizing these things. If you could - please just delete the firmware changes and move the json to the `nightmare` root directory. I plan on just removing the firmware directory - it was only there to make it easy for people before it was merged into upstream.

Again, thanks so much for your help!

**Review by cfbender — CHANGES_REQUESTED** [review]
This VIA json is different than the one that's in the VIA repo, this one doesn't work for me but the other one seems correct

**Comment by quark-works** [comment]
My bad! I had some files mixed up and forgot a comma here. Fixed it in the latest commit.

**Comment by cfbender** [comment]
@Armastardo if you could just move the json to the `nightmare/` root directory, and remove any changes other than the json, then I will get this merged!

**Review by cfbender — APPROVED** [review]

**Pull request merged**